business/checkers/authorization/principals_checker_test.go

```diff
@@ -0,0 +1,49 @@
+package authorization
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	security_v1beta "istio.io/client-go/pkg/apis/security/v1beta1"
+
+	"github.com/kiali/kiali/models"
+	"github.com/kiali/kiali/tests/data"
+	"github.com/kiali/kiali/tests/testutils/validations"
+)
+
+func TestPresentServiceAccount(t *testing.T) {
```

Comment:
Is it covered the case when an AuthorizationPolicy doesn't include any principal?